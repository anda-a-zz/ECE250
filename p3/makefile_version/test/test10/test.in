s -79.42;43.66
size
q_max 0.0;0.0;NW;p
q_min 0.0;0.0;SW;p
q_total 0.0;0.0;NE;p

