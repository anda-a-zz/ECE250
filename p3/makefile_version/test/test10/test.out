not found
tree size 0
failure
failure
failure
